Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -121,7 +121,12 @@ extension BrowserViewController: URLBarDelegate {
                     self.legacyShowSettings(deeplink: .contentBlocker)
                 }
             }
-
+            
+            if CoordinatorFlagManager.isEtpCoordinatorEnabled {
+                
+                self.navigationHandler?.showEnhancedTrackingProtection()
+            }
```

Comment:
We should put the code down here in the `else` case, so if flag is `true` then we only use coordinator